docs/subnets/customize-a-subnet.md

```diff
@@ -708,7 +826,7 @@ You can override these defaults with the following config:
 
 ### Fee Recipient
 
-This works together with [`allowFeeRecipients`](#setting-a-custom-fee-recipient) to specify where the fees should be sent to.
+This works together with [`allowFeeRecipients`](#setting-a-custom-fee-recipient) and [FeeRewardManager precompile](#distributing-fee-rewards) to specify where the fees should be sent to.
```

Comment:
#distributing-fee-rewards does not link to anywhere. Please correct.

Reply:
corrected